container/test.bzl

```diff
@@ -32,7 +34,8 @@ def _impl(ctx):
           "%{configs}": config_str,
           "%{workspace_name}": ctx.workspace_name,
           "%{test_executable}": ctx.executable._structure_test.short_path,
-          "%{image}": image_name,
+          # "%{image}": image_name,
```

Comment:
Remove commented code

Reply:
sorry, I pushed a WIP commit to this branch because I switched machines, should have let you know. fixed now